### PR TITLE
New nomis-data-hub account

### PR DESCRIPTION
**Pull request by SteveLinden**

### CODE REVIEW

File: environments/nomis-data-hub.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "account-type": "member",
+  "environments": [
+    {
+      "name": "development",
+      "access": [
+        {
+          "github_slug": "studio-webops",
+          "level": "sandbox",
+          "nuke": "exclude"
```
Comment (ewastempel) [inline environments/nomis-data-hub.json:10]: this should not be enabled by default

**Review by ewastempel — CHANGES_REQUESTED** [review]

File: environments/nomis-data-hub.json (hunk 1 above)
Comment (SteveLinden) [reply environments/nomis-data-hub.json:10]: Will remove the nuke

File: environments/nomis-data-hub.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      "access": [
+        {
+          "github_slug": "studio-webops",
+          "level": "developer"
+        }
+      ]
+    }
+  ],
+  "tags": {
+    "application": "nomis",
```
Comment (dms1981) [inline environments/nomis-data-hub.json:42]: Should this be `nomis-data-hub`?

Comment (SteveLinden) [reply environments/nomis-data-hub.json]: @dms1981 amended and pushed up again

**Review by dms1981 — APPROVED** [review]

**Review by ewastempel — APPROVED** [review]

**Pull request merged**